src/main/java/com/suse/saltstack/netapi/utils/ClientUtils.java

```diff
@@ -56,17 +56,25 @@ public static String streamToString(InputStream inputStream) {
     }
 
     /**
-     * Helper for constructing json object from kwargs and args.
+     * Helper for constructing json object with kwargs and args.
      *
      * @return JsonObject filled with kwargs and args.
      */
-    public static JsonObject makeJsonData(Map<String, String> kwargs, List<String> args) {
+    public static JsonObject makeJsonData(Map<String, String> props, Map<String, String> kwargs, List<String> args) {
```

Comment:
I actually hope we can merge #50 soon, and maybe this patch could be rebased on top afterwards. Then there is no way around satisfying checkstyle first.